### PR TITLE
Fix shortcuts detection on settings & game

### LINKED ISSUE #1493: Some hotkeys are bugged after last client update
some of the P/T setting hotkeys don't work, namely the altGr+ + and ctrl+ +, also the attach command ctrl+a doesn't work. this happened since the last client update about one week ago... also, trying to watch a replay crashes the program. Thank you for your attention!

### LINKED ISSUE #1499: Can't set certain keyboard shortcuts -- get "shortcut already in use" message
After playing around with adding more keyboard shortcuts, I found that I wanted to switch some (swapping modifier keys for my counters shortcuts). When I did this, I was only able to set one of the modifier keys for these shortcuts, and using the other claimed that the shortcut was already in use.

For reference, [here are all of my currently-set shortcuts as I can see them](http://imgur.com/muvP9IL,4LGxx2x,jichQMN,Q2A63xF). I used to have the following shortcuts in the "Counters" section:

* **Red**
  * Add: <kbd>Ctrl</kbd>+<kbd>,</kbd>
  * Remove: <kbd>Alt</kbd>+<kbd>,</kbd>
* **Green**
  * Add: <kbd>Ctrl</kbd>+<kbd>/</kbd>
  * Remove: <kbd>Alt</kbd>+<kbd>/</kbd>
* **Yellow**
  * Add: <kbd>Ctrl</kbd>+<kbd>.</kbd>
  * Remove: <kbd>Alt</kbd>+<kbd>.</kbd>

What is [currently shown in this screenshot](http://i.imgur.com/Q2A63xF.png) is what I ended up with after removing all of these shortcuts and attempting to swap them. (<kbd>Ctrl</kbd> for removing, <kbd>Alt</kbd> for adding)

This behavior was observed on Windows 7 using 60ed9fd.

### PR DESCRIPTION
**Pull request by marcofernandezheras**
This should fix 
Fix #1499
Fix #1493
not sure about #1477

With this we can bind a single key to shortcut too

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Can somebody other than me test this? I wasn't experiencing the bug, so I'm not really certain what to look for.

File: cockatrice/src/shortcutssettings.cpp [hunk 1]
```diff
@@ -100,7 +100,7 @@ bool ShortcutsSettings::isValid(QString name, QString secuences)
 {
     QString checkKey = name.left(name.indexOf("/"));
 
-    QStringList stringSecuences = secuences.split(";");
```
Comment (Daenyth) [inline cockatrice/src/shortcutssettings.cpp]: "Sequence", not "secuence"

**Comment by Daenyth** [comment]
Can you amend the commit message to describe the issue that it fixes, maybe include the ticket numbers in the commit message so they get linked?

**Comment by ZeldaZach** [comment]
So is this good to go @Daenyth ?

**Comment by Daenyth** [comment]
If it solves the problem. I wasn't hitting it.

It looks harmless enough.

**Pull request merged**